middleman_protocol/setup.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+from setuptools import setup
+
+
+setup(
+    name='Middleman-Protocol',
+    version='0.1',
+    url='',
+    maintainer='',
+    maintainer_email='',
+    packages=[
+        'middleman_protocol',
+    ],
+    package_data={},
+    python_requires='>=3.6',
+    install_requires=[],
```

Comment:
List dependencies too. You can remove them from service's `requirements.txt`.

My idea is for this to be a library that's installed in the virtualenv along with other dependencies. Concent and Signing Service then can both list this library as their dependency.

Reply:
:+1: done